tox.ini

```diff
@@ -28,9 +28,9 @@ setenv =
     zeo4: ZEO4_SERVER=1
 commands =
     # Run unit tests first.
-    zope-testrunner -u --test-path=src {posargs:-vc}
+    zope-testrunner -u --test-path=src --all {posargs:-vc}
```

Comment:
Apparently, `--all` takes precedence over `-a`. This would imply that the expensive tests cannot be avoided for `tox` runs.

I see two potential workarounds:
1. use `-a 0` instead of `-all`; because the argument parser allows multiple `-a` options and takes the last value, the `-a 0` from `tox.ini` could be overridden with a `-a 1`
2. move the `-all` into the `posarg` default; this assumes that in `{`*var*`:`*xxx*`}`, the *xxx* specifies the default to be used if *var* is empty.

Reply:
@d-maurer thanks for feedback and good point. I believe the first option is better, because on the second on, if _any_ positional argument is used, then the default settings for level will be deactivated. The only difference is that I used `-a 1000` instead of `-a 0`, because contrary to what zope.testrunner documentation says, `-a 0` actually _disables_ tests at all levels. Please see https://github.com/zopefoundation/ZEO/pull/210/commits/06f8e1461c05b3664512eac533947fd38d266946 for details. Hope it is ok.

P.S. after settling here we should adjust .meta and tox.ini in ZODB.git correspondingly.